scripts/install_submodules.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/set_vars.sh"
```

Comment:
What's the point of sourcing it unconditionally? That would override all the settings that a user might've exported before, for example, branches of the sc-machine or sc-web.